types/header.go

```diff
@@ -69,3 +69,41 @@ func (b *BlockNumber) Decode(decoder scale.Decoder) error {
 	*b = BlockNumber(u.Uint64())
 	return err
 }
+
+type OptionBlockNumber struct {
```

Comment:
Doesnt it make sense to use the Option[BlockNumber] generic here?

Reply:
Yes, forgot to clean this up.

Reply:
Removed.